source/nodejs/aclint/tslint.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
This should probably not be part of this project -- need to finalize move to eslint